src/components/Pages/ManageWiki/Cards/Logo.vue

```diff
@@ -14,7 +14,12 @@
       <v-img v-if="logo" :src="logo" max-width="135" />
     </v-card-text>
     <v-card-actions>
-      <v-btn :disabled="this.selectedLogoFile === null" @click="doLogoUpload">Set Logo</v-btn>
+      <v-tooltip top>
+        <template v-slot:activator="{ on, attrs }">
+        <v-btn :disabled="this.selectedLogoFile === null" v-bind="attrs" v-on="on" @click="doLogoUpload">Set Logo</v-btn>
```

Comment:
Getting `Error in render: "TypeError: Cannot read property 'selectedLogoFile' of null`. Fixed by:

```suggestion
        <v-btn :disabled="selectedLogoFile === null" v-bind="attrs" v-on="on" @click="doLogoUpload">Set Logo</v-btn>
```